.github/workflows/find_duplicate_issue.yml

```diff
@@ -0,0 +1,78 @@
+name: Find duplicate from issue id
+
+on:
+  workflow_dispatch:
+    inputs:
+      issue_id:
+        description: 'Id of the issue to check for duplicates'
+        required: true
+
+jobs:
+  deduplicate:
+    runs-on: ubuntu-latest
+    timeout-minutes: 10
+    permissions:
+      contents: read
+      issues: write
```

Comment:
~~Does it need the write permission on gh issues?~~

Edit: yes, Claud will comment/manage the duplicates